pkg/codegen/java/gen_program.go

```diff
@@ -305,7 +305,7 @@ func GenerateProject(
 	}
 
 	repositories := make(map[string]bool)
-	coreSDKVersion := "(,1.0]"
+	coreSDKVersion := DefaultSdkVersion.String()
```

Comment:
Out of curiosity I looked at https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges and this will now become a "soft" requirement of "1.0.0"

> Default strategy: Of the overlapping ranges, the highest soft requirement is the version to be used.

Might be worth linking to the doc here.

Reply:
Good shout -- have added a link to the Maven docs along with a comment. FWIW I expect us to change this soon to a range, perhaps with 1.0 as the lower bound and HEAD as the upper, maybe.